pkg/logql/engine_test.go

```diff
@@ -676,6 +680,272 @@ func TestEngine_NewRangeQuery(t *testing.T) {
 				},
 			},
 		},
+		// binops
+		{
+			`
+rate({app="foo"}[1m]) or
```

Comment:
Can you move all of those to the line above ? Or just follow the same convention as in line 741.